src/api/firebase.js

```diff
@@ -170,25 +171,23 @@ export async function shareList(listPath, currentUserId, recipientEmail) {
  */
 export async function addItem(listPath, { itemName, daysUntilNextPurchase }) {
 	const listCollectionRef = collection(db, listPath, 'items');
-	// TODO: Replace this call to console.log with the appropriate
-	// Firebase function, so this information is sent to your database!
 	return addDoc(listCollectionRef, {
 		dateCreated: new Date(),
-		// NOTE: This is null because the item has just been created.
-		// We'll use updateItem to put a Date here when the item is purchased!
 		dateLastPurchased: null,
 		dateNextPurchased: getFutureDate(daysUntilNextPurchase),
 		name: itemName,
 		totalPurchases: 0,
 	});
 }
 
-export async function updateItem() {
-	/**
-	 * TODO: Fill this out so that it uses the correct Firestore function
-	 * to update an existing item. You'll need to figure out what arguments
-	 * this function must accept!
-	 */
+export async function updateItem(listPath, itemId) {
+	const itemDocRef = doc(db, listPath, 'items', itemId);
+	const datePurchased = new Date();
+	await updateDoc(itemDocRef, {
+		dateLastPurchased: datePurchased,
+		totalPurchases: increment(1),
+		dateNextPurchased: getFutureDate(1),
```

Comment:
It looks like `dateNextPurchased` isn't being utilized correctly in this context! According to the issue notes, we don't have to worry about the `dateNextPurchased` at this stage as this will be handled in a separate issue!

Sneak peak: The acceptance criteria in issue 11 specify that `dateNextPurchased` should be calculated using the `calculateEstimate` function and then saved to the Firestore database!